selectors/views/channel_sidebar.ts

```diff
@@ -211,28 +214,32 @@ function maxDefined(a: number, b?: number) {
     return typeof b === 'undefined' ? a : Math.max(a, b);
 }
 
+function concatChannels(channelsA: Channel[], channelsB: Channel[]) {
+    return [...channelsA, ...channelsB];
+}
+
+const memoizedConcatChannels = memoizeOne(concatChannels);
```

Comment:
What's the difference between `memoizeOne` and the old `memoizeResult` function? Comparing the source for that, they look mostly the same except `memoizeOne` is spreading the arguments array which is supposedly worse according to the comments that I copied from `reselect` at some point. I don't know if those comments are still relevant since those comments are rather old, but I don't know if either would perform better or worse anyway.

Also, declaring this out here isn't great because it could allow others to accidentally use it and overwrite the memoized array. The reason it was written kind of weirdly before was to hide `memoizedConcat` from other functions in this file so that it couldn't be accidentally misused

Reply:
replaced it with `memoizeResult`